client/profile-wizard/steps/product-types/index.js

```diff
@@ -201,6 +239,22 @@ export class ProductTypes extends Component {
 							'woocommerce-admin'
 						) }
 					</Text>
+					{ window.wcAdminFeatures &&
+						window.wcAdminFeatures.subscriptions &&
+						! installedPlugins.includes( 'woocommerce-payments' ) &&
```

Comment:
This is kinda minimal, but now with the `promise.all` this footnote will quickly disapear just before it moves to the next page. Maybe there is way to get `installedPlugins` only on first render and not update it afterwards?

Reply:
Thank you @louwie17. If it's ok for you, I'm going to merge this PR and tackle that issue in a follow-up PR.

Reply:
Yeah that's good with me @octaedro